app/assets/javascripts/discourse/models/topic.js.es6

```diff
@@ -117,6 +117,7 @@ const Topic = RestModel.extend({
       const titleDir = isRTL(title) ? "rtl" : "ltr";
       return `<span dir="${titleDir}">${fancyTitle}</span>`;
     }
+    fancyTitle = fancyTitle + (bookmark_name ? " (" + bookmark_name + ")" : "");
```

Comment:
not very readable, might need intermediate variable

Reply:
How about this?

```js
if (bookmark_name) fancyTitle += ` (${bookmark_name})`;
```